tsconfig.build.json

```
@@ -3,7 +3,10 @@
 		"declaration": true,
 		"noEmit": false,
 		"outDir": "dist",
-		"rootDir": "src"
+		"rootDirs": [
+			"../../vendor",
+			"src",
+		  ]
 	},
 	"extends": "./tsconfig.json",
 	"include": ["src"],
```

i think `vendor` needs to go in `include` rather than rootDirs
```suggestion
		"rootDirs": ["src"]
	},
	"extends": "./tsconfig.json",
	"include": ["src", "vendor"],
```

---

Tried that but it brought back the same build errors 😞

`Error: src/third-party-tags/lotame.ts(1,30): error TS6059: File '/home/runner/work/commercial-core/commercial-core/vendor/lotame-script.ts' is not under 'rootDir' '/home/runner/work/commercial-core/commercial-core/src'. 'rootDir' is expected to contain all source files.
Error: src/third-party-tags/twitter-uwt.ts(1,48): error TS6059: File '/home/runner/work/commercial-core/commercial-core/vendor/twitter-script.ts' is not under 'rootDir' '/home/runner/work/commercial-core/commercial-core/src'. 'rootDir' is expected to contain all source files.`

---

oh sorry!